app/validators/xml_validator.rb

```diff
@@ -2,9 +2,12 @@
 
 class XmlValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
-    xml_errors = xsd(record).validate(Nokogiri::XML(value))
+    xml = Nokogiri::XML(value) { |config| config.strict }
+    xml_errors = xsd(record).validate(xml)
```

Comment:
should I keep this as a single statement? I originally split it into two when I was going to target the rescue to the xml parsing only, and not to the XSD validation

Reply:
No, I like this better as two lines.